scripts/install_script_ubuntu.sh

```diff
@@ -149,6 +149,7 @@ sudo apt-get install -y \
   libfreetype6-dev \
   libpng-dev \
   libprotobuf-dev \
+  libmysqlclient-dev \
```

Comment:
This also needs to go into https://github.com/google/grr/blob/master/travis/before_install.sh